### DISCUSSION TITLE
Unable to use ssh-copy-id

### DISCUSSION THREAD

**Discussion by mcuix** [blinksh/blink#1538] [Support] [answered]
I am unable to ssh-copy-id my ssh keys to my server.
I use 

> ssh-copy-id /.ssh/id_rsa.pub user@myserver

, but Blink returns an **ERROR no identity found**.

**Comment by yury** [comment]
Hi @mcuix,

‘ssh-copy-id’ command works only with keys from config->keys for now.

**Comment by JeremyIglehart** [comment, marked as answer]
Okay, it took me a while to figure this out because I couldn’t find any how to... maybe there is one somewhere and I overlooked it... if so, please feel free to point this out to us all.

Basically, I just installed Blink yesterday - my version comes with a generated key “id_rsa”.  Assuming you don’t change this key, or make a new one with a different name.  You could do something like:

```
ssh-copy-id id_rsa jeremy@192.168.10.10
```

Obviously you should use your username and host... but the ssh-copy-id is looking for the names of keys listed when you type config, and choose “keys” out of the menu.

Enjoy.

**Comment by ellierider** [comment]
> Hi @mcuix,
> 
> ‘ssh-copy-id’ command works only with keys from config->keys for now.

Can you please provide an example?

**Comment by carloscabanero** [comment]
There is more information here as well. The key is not a path but a key on your config > keys.

https://docs.blink.sh/basics/commands#ssh-copy-id

**Reply by kyleridolfo** [reply]
Thanks. I tried what @JeremyIglehart did, and looked at that link. But it simply does not find either type of key I created with the app. I tried a regular id_rsa key and I tried the Secure Enclave version

**Reply by carloscabanero** [reply]
What error message do you get? In any case, the only thing this script does is to copy the public key to the "authorized_keys" on the remote. If you still have issues, you can do it manually as well by logging to the remote.

**Comment by kyleridolfo** [comment]
Hi! I was able to get it working on iPhone. On iPad it just said it
couldn’t find the key
